common/components/componentsBySection/FindProjects/ProjectCardsContainer.jsx

```diff
@@ -100,6 +103,14 @@ class ProjectCardsContainer extends React.Component<Props, State> {
   }
 
   _renderCards(): React$Node {
+    if (
+      this.state.error ||
+      (this.state.projects &&
+        this.state.projects.size === 0 &&
```

Comment:
```suggestion
      (_.isEmpty(this.state.projects) &&
```

Reply:
After testing, if change it like this, it will always hide the component